src/components/app-configure/store/UserLimits.vue

```diff
@@ -0,0 +1,234 @@
+                  class="usage-unit-dropdown"
+                  trigger-class="usage-unit-dropdown-trigger"
+                  :disabled="storageLimitKind === 'Unlimited'"
+                  @update:model-value="
+                    handleUserLimitFieldChange({
+                      type: 'storage',
+                      field: 'unit',
+                      value: $event,
+                    })
+                  "
+                />
+              </div>
+            </VStack>
+          </VStack>
+        </VStack>
+        <VStack direction="column" gap="0.75rem" align="start">
+          <h4>Bandwidth</h4>
+          <VStack gap="1.25rem" wrap class="width-100">
+            <VDropdown
+              :options="userLimitOptions"
+              :model-value="bandwidthLimitKind"
+              class="limits-type-dropdown"
+              trigger-class="limits-type-dropdown-trigger"
+              @update:model-value="handleIsUnlimitedChange('bandwidth', $event)"
+            />
+            <VStack direction="column" class="flex-grow">
+              <div class="input-group">
+                <VTextField
+                  id="bandwidth-user-limit"
+                  type="number"
+                  min="1"
+                  max="1024"
+                  :disabled="bandwidthLimitKind === 'Unlimited'"
+                  :model-value="bandwidthLimit.value"
+                  no-message
+                  class="usage-value-textfield"
+                  placeholder="value"
+                  @update:model-value="
+                    handleUserLimitFieldChange({
+                      type: 'bandwidth',
+                      field: 'value',
+                      value: $event,
+                    })
+                  "
+                />
+                <VDropdown
+                  :options="bandwidthUnits"
+                  :model-value="bandwidthLimitUnit"
+                  display-field="label"
+                  placeholder="unit"
+                  class="usage-unit-dropdown"
+                  trigger-class="usage-unit-dropdown-trigger"
+                  :disabled="bandwidthLimitKind === 'Unlimited'"
+                  @update:model-value="
+                    handleUserLimitFieldChange({
+                      type: 'bandwidth',
+                      field: 'unit',
+                      value: $event.value,
+                    })
+                  "
+                />
+              </div>
+            </VStack>
+          </VStack>
+        </VStack>
+      </VStack>
+    </SettingCard>
+  </section>
+</template>
+
+<style scoped>
+.limits-type-dropdown {
+  min-width: 12rem;
+}
+
+.input-group {
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.usage-value-textfield {
+  width: calc(100% - 10rem);
+}
+
+.usage-unit-dropdown {
+  min-width: 10rem;
+}
+
+@media screen and (min-width: 1024px) {
+  .limits-input-container > * {
+    flex: 0 0 calc(50% - 1rem);
+  }
+}
+</style>
+
+<style>
```

Comment:
why not move this to scoped style, I guess `!important` is not needed when moved.

Reply:
Actually it won't work in scoped style. Dropdown trigger is not part of this component but is part of Dropdown component and to style that we'll either need to style it in Dropdown component (where we can use scoped style without `!important` flag) or we'll need to do it globally or as an unscoped internal style with `!important` flag to override the defaults.